common/performance.py

```
@@ -54,7 +54,7 @@ def as_view(cls, **initkwargs) -> Callable[..., HttpResponse]:
         """
         Returns a view function for the template view with caching enabled.
         """
-        return cache_page(int(os.environ["CACHE_TTL"]))(
+        return cache_page(int(os.environ.get("CACHE_TTL", "86400")))(
             super(CachedTemplateView, cls).as_view(**initkwargs)
```

Ruff (0.15.14)

[warning] Use `super()` instead of `super(__class__, self)`

Remove `super()` parameters

(UP008)